### PR TITLE
Create monitoring synth script and regenerate gapic

### PR DESCRIPTION
**Pull request by dazuma**
Of note: the new gapic requires a just-released update to googleapis-common-protos-types.

### CODE REVIEW

File: google-cloud-monitoring/lib/google/cloud/monitoring.rb [hunk 1]
```diff
@@ -170,7 +167,7 @@ def self.new(*args, version: :v3, **kwargs)
       module Group
         ##
         # The Group API lets you inspect and manage your
-        # [groups](https://cloud.google.comgoogle.monitoring.v3.Group).
+        # [groups](https://cloud.google.com#google.monitoring.v3.Group).
```
Comment (blowmage) [inline google-cloud-monitoring/lib/google/cloud/monitoring.rb]: Is this URL correct?

Comment (dazuma) [reply google-cloud-monitoring/lib/google/cloud/monitoring.rb]: No it's not, but it's better than what was there before. I need to track that one down internally.

**Comment by blowmage** [comment]
Is the updated googleapis-common-protos-types dependency just for this gem or will it be applied across the board?

**Comment by dazuma** [comment]
The updated dependency is required for the current logging and monitoring generated code. It's not (currently) required for any of the other libraries.

**Pull request merged**